core/controllers/editor.py

```diff
@@ -687,7 +687,7 @@ def post(self, exploration_id):
         self.render_json({})
 
 
-class ExplorationStatisticsHandler(EditorHandler):
+class ExplorationStatisticsHandlerOld(EditorHandler):
```

Comment:
OldExplorationStatisticsHandler. Our controller names end in Handler.

Reply:
Done.